mentat/commands.py

```diff
@@ -390,3 +388,18 @@ def argument_names(cls) -> list[str]:
     @classmethod
     def help_message(cls) -> str:
         return "Set a configuration option or omit value to see current value."
+
+
+class RunCommand(Command, command_name="run"):
+    async def apply(self, *args: str) -> None:
+        session_context = SESSION_CONTEXT.get()
+        conversation = session_context.conversation
+        await conversation.run_command(list(args))
+
+    @classmethod
+    def argument_names(cls) -> list[str]:
+        return ["command"]
+
+    @classmethod
+    def help_message(cls) -> str:
+        return "Run a shell command and send the output the model's conversation."
```

Comment:
I'd prefer "Run a shell command and put its output in context.". (Also note there's a typo)